src/translators/rust/pure/cargo-lock/default.nix

```diff
@@ -0,0 +1,50 @@
+{
+  externals,
+  translatorName,
+  utils,
+  lib,
+  runCommandNoCC,
+  ...
+}:
+
+let
+  translate =
+    {
+      inputPaths,
+      ...
+    }:
+    let
+      cargoLock = builtins.fromTOML (builtins.readFile (builtins.elemAt inputPaths 0));
```

Comment:
@DavHau just thinking out loud here, but I'm thinking that it might make sense to have inputPaths be a dictionary with name keys ? (rather than a list). Relying on the position of an attribute could be error prone ?
Just a thought.

Reply:
Actually I'm currently thinking about removing the path list and instead only allow to pass one path.
One reason is that pure translators should be callable via nix expression, and there it is simple to specify just one source.
If the source needs extra files, then it can be patched inside the nix expression before handing it to the translator